status/missing_pods.go

```diff
@@ -22,28 +22,60 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
+// DefaultClientPool is a key in the NodeCountByPool map on the LoadTestMissing
+// struct. It maps to the number of nodes required from the default client pool.
+const DefaultClientPool = "default-client-pool"
+
+// DefaultDriverPool is a key in the NodeCountByPool map on the LoadTestMissing
+// struct. It maps to the number of nodes required from the default driver pool.
+const DefaultDriverPool = "default-driver-pool"
+
+// DefaultServerPool is a key in the NodeCountByPool map on the LoadTestMissing
+// struct. It maps to the number of nodes required from the default server pool.
+const DefaultServerPool = "default-server-pool"
+
 // LoadTestMissing defines missing pods of LoadTest.
 type LoadTestMissing struct {
 	// Driver is the component that orchestrates the test. If Driver is not set
 	// that means we already have the Driver running.
-	Driver *grpcv1.Driver `json:"driver,omitempty"`
+	Driver *grpcv1.Driver
 
 	// Servers are a list of components that receive traffic from. The list
 	// indicates the Servers still in need.
-	Servers []grpcv1.Server `json:"servers,omitempty"`
+	Servers []grpcv1.Server
 
 	// Clients are a list of components that send traffic to servers. The list
 	// indicates the Clients still in need.
-	Clients []grpcv1.Client `json:"clients,omitempty"`
+	Clients []grpcv1.Client
+
+	// NodeCountPyPool is a map which gives the number of nodes required from
+	// each pool to run the test. The names of the required node pools are the
+	// keys and the value is the number of nodes required from the named pool.
+	// If a test does not require any nodes from a pool, the pool name will not
+	// be present as a key in the map.
+	NodeCountByPool map[string]int
```

Comment:
TODO: It may be a good idea to add some test cases for default pools. NTS: Need to figure out how to structure them, since they would divert from the Context-blocks that are currently in place. Ideas? @wanlin31